searchresult.py

```diff
@@ -77,15 +83,21 @@ def analyze(self, fragment: Optional[bool] = False) -> lal.AnalysisUnit:
         :return: An analysis unit containing the tree
         :rtype: class:'libadalang.AnalysisUnit'
```

Comment:
Please discuss how to prevent duplication:
in the signature is already written " -> lal.AnalysisUnit:", so repeating this might be unneeded extra effort.
Especially, since it make me wonder is 'libadalang.AnalysisUnit' something else than 'lal.AnalysisUnit'